engine/pokedex/pokedex_2.asm

```diff
@@ -148,11 +150,12 @@ DisplayDexEntry:
 	or d
 	jr z, .skip_weight
 	push de
+; Print the height with four of the five digits will be in front of the decimal point
```

Comment:
Grammar: "Print the height, with four of the five digits in front of the decimal point."